skyportal/models.py

```diff
@@ -3088,6 +2666,7 @@ class FacilityTransaction(Base):
 
 
 class Listing(Base):
+    update = delete = AccessibleIfUserMatches("user")
```

Comment:
Should this apply to the `read` permissions as well? That's what seems like is the expected behavior of the current Listing logic